### DISCUSSION TITLE
Windows 7 / Vista Task Bar.

### DISCUSSION THREAD

**Discussion by Espaece** [valinet/ExplorerPatcher#2618] [Ideas]
There isn't much for me to say but if this was a feature, people would really love it.
![d1yf7vw-a0017c23-27d5-45d6-8b42-71e421ad0d69](https://github.com/valinet/ExplorerPatcher/assets/154742748/e5838322-d25d-4b88-9232-0f8394ea8a91)

**Comment by pyrates999** [comment]
That code was removed when windows 8 was released.  So this can't be brought back unfortunately.

https://answers.microsoft.com/en-us/windows/forum/all/why-would-microsoft-remove-the-beautiful-aero/d924e3c7-3f4f-4bcc-ae37-ad7027ad0499
